src/pages/questionPage.js

```diff
@@ -93,7 +93,7 @@ const selectAnswer = (event) => {
 
      const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
 
-     showInformation(currentQuestion.text)
+     showInformation(currentQuestion)
```

Comment:
This makes it so the information text isn't displaying properly

Reply:
Yep, I now fix the bag